style.css

```diff
@@ -0,0 +1,328 @@
+* {
+    box-sizing: border-box;
+    font-family: 'Lato', sans-serif;
+  }
+  body{
+      background-color: #ECF0F1;
+      color: #2C3E50;
+  }
+  .content{
+    max-width: 90%;
+    margin: 0 auto;
+    /* padding-left: 20px; */
+  }
+
+  body a{
+      color: #2C3E50;
+      text-decoration: none;
+  }
+  hr{
+      border: 1px solid #fc7e00;
+  }
+  h1{
+    color:white;
+    /* display: inline-block; */
+    /* vertical-align: top; */
+    float: left;
+    width: 240px;
+    padding: .5rem;
+  }
+  h2{
+      font-size: 2rem;
+  }
+  h3{
+    font-size: 1.5rem;
+  }
+  h4{
+    font-size: 1.75rem;
+  }
+  h5{
+    font-weight: bold;
+    font-size: 1.15rem;
+    line-height: 1.25rem;
+    margin: .75rem 0;
+  }
+  .header__image {
+    width: 60px;
+    margin: 1.25rem 0 0 1rem;
+    float: left;
+  }
+  header{
+      background-color: #112d49;
+  }
+  img{
+      max-width: 100%;
+  }
+
+  nav{
```

Comment:
This should really use a class. You may have more than one `nav` tag in your HTML.